grammar.peg

```diff
@@ -134,9 +150,22 @@ IndexExpression "index" <- "[" _? lit:StringLiteral _? "]" {
    return false, errors.New("Unclosed index expression")
 }
 
-Value "value" <- selector:Selector { return &MatchValue{Raw:strings.Join(selector.(Selector), ".")}, nil }
-       / n:NumberLiteral { return &MatchValue{Raw: n.(string)}, nil }
-       / s:StringLiteral { return &MatchValue{Raw: s.(string)}, nil}
+Value "value" <- selector:Selector {
+   var raw string
+   sel := selector.(Selector)
+   switch {
+	case len(sel) == 0:
+	case sel[0] == "/":
+		raw = strings.Join(sel, "/")
+	default:
+		raw = strings.Join(sel, ".")
+   }
+   return &MatchValue{Raw:raw}, nil 
```

Comment:
```suggestion
   return &MatchValue{Raw:selector.(Selector).String()}, nil 
```

Could this work?

Reply:
Oh, yeah. I just followed the model there before rather blindly. :-)